SchulichDeltaHermes.pro

```diff
@@ -1,38 +1,53 @@
+
 #-------------------------------------------------
 #
-# Project created by QtCreator 2014-10-13T12:01:32
+#  Project created by QtCreator 2014-10-13T12:01:32
+#
+#  Schulich Delta Hermes
+#  Copyright (C) 2015 University of Calgary Solar Car Team
+#
+#  This file is part of Schulich Delta Hermes
+#
+#  Schulich Delta Hermes is free software: 
+#  you can redistribute it and/or modify it under the terms 
+#  of the GNU Affero General Public License as published by 
+#  the Free Software Foundation, either version 3 of the
+#  License, or (at your option) any later version.
+#
+#  Schulich Delta Hermes is distributed 
+#  in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+#  without even the implied warranty of MERCHANTABILITY or 
+#  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero 
+#  General Public License for more details.
+#
+#  You should have received a copy of the GNU Affero General 
+#  Public License along with Schulich Delta Hermes.
+#  If not, see <http://www.gnu.org/licenses/>.
+#
+#  For further contact, email <software@calgarysolarcar.ca>
 #
 #-------------------------------------------------
 
-QT       += core gui serialport
+QT += core gui serialport
 
-greaterThan(QT_MAJOR_VERSION, 4): QT += widgets webkitwidgets
+greaterThan(QT_MAJOR_VERSION, 4): QT += widgets network
 
-TARGET = ../release/SchulichDeltaDesktopTelemetry
+TARGET = ../release/SchulichDeltaHermes
 TEMPLATE = app
 CONFIG += static
 
-! include ( /usr/local/qwt-6.1.2/features/qwt.prf ) {
-   error("Can not find qwt.prf!" )
-}
-
 QMAKE_CXXFLAGS +=
```

Comment:
I am still a strong supporter of the -Wall and -Werror flags. I think it forces better code and I have seen it prevent so many bugs before.

Something to think about doing for new projects. Always better to do it sooner than later.